app/src/main/res/drawable/cloudly.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="32dp" android:viewportHeight="512.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.cloudly` appears to be unused
 Unused resources make applications larger and slow down builds.